src/Microsoft.Net.Http.Headers/MediaTypeHeaderValue.cs

```diff
@@ -77,6 +100,9 @@ public StringSegment Charset
             }
         }
 
+        /// <summary>
+        /// Gets the <see cref="System.Text.Encoding"/> of the <see cref="MediaTypeHeaderValue"/> if it has one.
+        /// </summary>
```

Comment:
This is the encoding that *corresponds* to the value of the charset parameter. This should probably point to `Encoding.WebName` to explain that's how they are associated.